docs/content/installation/installation-with-helm.md

```diff
@@ -230,6 +230,8 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 |``controller.readyStatus.enable`` | Enables the readiness endpoint `"/nginx-ready"`. The endpoint returns a success code when NGINX has loaded all the config after the startup. This also configures a readiness probe for the Ingress Controller pods that uses the readiness endpoint. | true |
 |``controller.readyStatus.port`` | The HTTP port for the readiness endpoint. | 8081 |
 |``controller.enableLatencyMetrics`` | Enable collection of latency metrics for upstreams. Requires ``prometheus.create``. | false |
+``controller.minReadySeconds`` | Specifies the minimum number of seconds for which a newly created Pod should be ready without any of its containers crashing, for it to be considered available. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) | 0 |
+``controller.strategy`` | Specifies the strategy used to replace old Pods by new ones. [docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) | {} |
```

Comment:
Looks like there are some `|` missing at the beginning of the lines

Reply:
Thanks. Added.